### DISCUSSION TITLE
Can I use C++20

### DISCUSSION THREAD

**Discussion by Dibyojyo** [slycelote/caide#44] [Q&A] [answered]
I had a question, I used C++20, in the caide.ini file and parsed some problems, it didn't work, How to resolve this ??🤔🤔

**Comment by slycelote** [comment, marked as answer]
What exactly didn't work? What was the error message?

**Reply by Spidy104** [reply]
I use c++17, never use c++20 it works fine, btw vs caide doesn't show time running per test...

**Reply by slycelote** [reply]
Try creating a new caide solution
